src/types.h

```diff
@@ -93,6 +94,13 @@ struct StackType
   }
 };
 
+enum class TimestampMode
```

Comment:
```suggestion
enum class TimestampMode : uint8_t
```
Nit: I updated the other enums in this file to keep the size of SizedType down, so let's do the same to this one